src/modules/db_redis/db_redis_mod.c

```diff
@@ -29,6 +29,12 @@
 #include "redis_dbase.h"
 #include "redis_table.h"
 
+#ifdef WITH_SSL
+int db_redis_opt_tls = 0;
+char *ca_path = 0;
+#endif
+char *db_pass = 0;
```

Comment:
Add module name prefix to the global variables ca_path and db_pass, like it is db_redis_opt_tls.  The are also declared with extern and being prefixed reduce the chances of conflicts with similar variables (e.g., ca_path can be quite common with some libs).

Reply:
Very good point! Thank you very much. 

Done.